Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Build
+FROM rust:1.73-alpine3.18 AS build
+
+WORKDIR /grafbase
+
+COPY ./cli ./cli
+COPY ./engine ./engine
+
+WORKDIR /grafbase/cli
+
+RUN apk add --no-cache git musl-dev
+
+RUN cargo build --release
+
+# Run
+FROM alpine:3.18
+
+WORKDIR /grafbase
+
+RUN apk add --no-cache nodejs npm
+
+COPY --from=build /grafbase/cli/target/release/grafbase /bin/grafbase
+
+ENTRYPOINT ["/bin/grafbase"]
```

Comment:
We should not run as root. Instead we should create a user under which the cli runs.

Reply:
Example?